### DISCUSSION TITLE
Supabase for Location services

### DISCUSSION THREAD

**Discussion by thecybergypsy** [supabase/supabase#14541] [Questions] [answered]
I'm just tossing up which service to use as a backend for an on-demand labour website that would require strong geolocation features, I love the whole Supabase project, but am also torn to MongoDB, any advice here you wonderfully seasoned Supa folks..

**Comment by GaryAustin1** [comment, marked as answer]
Supabase has all the features of Postgres and has the Postgis extensions which are quite extensive.
https://postgis.net/documentation/faq/
